bin/ch/$262.js

```diff
@@ -0,0 +1,24 @@
+R"(
```

Comment:
actually, I just checked we are supporting VS2013 and VS2015, both support c++11 raw string.
so no pre-build is necessary.
see https://msdn.microsoft.com/en-us/library/69ze775t(v=vs.120).aspx